logstash-core.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = "http://www.elastic.co/guide/en/logstash/current/index.html"
   gem.license       = "Apache License (2.0)"
 
-  gem.files         = Dir.glob(["lib/**/*.rb", "locales/*"])
+  gem.files         = Dir.glob(["lib/logstash-core.rb", "lib/logstash/**/*.rb", "spec/**/*.rb", "locales/*"])
```

Comment:
I see your point. However I still have question, do we still want/need ```lib\logstash-event.rb``` ? might be we can get rid of this too.